src/aws/ecs.tf

```diff
@@ -0,0 +1,96 @@
+resource "aws_ecs_cluster" "marsha_ffmpeg_transmux_cluster" {
+  name = "${terraform.workspace}-marsha"
+}
+
+resource "aws_ecs_service" "marsha_ffmpeg_transmux_service" {
+  name            = "${terraform.workspace}-marsha-ffmpeg-transmux-service"
+  cluster         = aws_ecs_cluster.marsha_ffmpeg_transmux_cluster.id
+  task_definition = aws_ecs_task_definition.marsha_ffmpeg_transmux_definition.arn
+  launch_type     = "FARGATE"
+
+  network_configuration {
+    subnets = [ 
+      aws_subnet.fargate_ffmpeg_transmux_vpc_public_subnet1.id,
+      aws_subnet.fargate_ffmpeg_transmux_vpc_public_subnet2.id 
+    ]
+    security_groups = [ aws_security_group.fargate_ffmpeg_transmux_security_group.id ]
+    assign_public_ip = true
+  }
+
+  depends_on = [aws_iam_role.ecs_task_ffmpeg_transmux_execution_role]
+}
+
+resource "aws_ecs_task_definition" "marsha_ffmpeg_transmux_definition" {
+  family = "${terraform.workspace}-marsha-ffmpeg-transmux-definition"
+  container_definitions = <<TASK_DEFINITION
+[
+  {
+    "name": "${terraform.workspace}-marsha-ffmpeg-transmux",
+    "dnsSearchDomains": null,
+    "environmentFiles": null,
+    "logConfiguration": {
+      "logDriver": "awslogs",
+      "secretOptions": null,
+      "options": {
+        "awslogs-group": "/ecs/${terraform.workspace}-fargate-ffmpeg-transmux",
+        "awslogs-region": "eu-west-1",
+        "awslogs-stream-prefix": "ecs"
+      }
+    },
+    "entryPoint": [],
+    "portMappings": [
+      {
+        "hostPort": ${var.ffmpeg_transmux_container_port},
+        "protocol": "tcp",
+        "containerPort": ${var.ffmpeg_transmux_container_port}
+      }
+    ],
+    "command": [],
+    "linuxParameters": null,
+    "cpu": 0,
+    "environment": [
+      {
+        "name": "DESTINATION_BUCKET_NAME",
+        "value": "${aws_s3_bucket.marsha_destination.bucket}"
+      }
+    ],
+    "resourceRequirements": null,
+    "ulimits": null,
+    "dnsServers": null,
+    "mountPoints": [],
+    "workingDirectory": null,
+    "secrets": null,
+    "dockerSecurityOptions": null,
+    "memory": null,
+    "memoryReservation": ${var.ffmpeg_transmux_container_memory_reservation},
+    "volumesFrom": [],
+    "stopTimeout": null,
```

Comment:
Maybe you want to add a big stopTimeout as a safeGuard ?

Reply:
I have no idea the value to use. 1 hour ?

Reply:
I don't know how much time it will take to process a very long livestream recording (and if there is a time limit for them).
But I'm thinking something like 4 hours, just to be sure to kill expensive zombie tasks.

Do you know what happens if a job fails (return code != 0 or timeout) ? Is there a possibility to be notified ?

Reply:
> Do you know what happens if a job fails (return code != 0 or timeout) ? Is there a possibility to be notified ?

I have to work on this part. When a task fail an event can be emitted, this event will trigger a lambda we defined and in this lambda we have to determine what to do. Yet Another Lambda...